test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ContextTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Health.Fhir.Core.Features;
+using Microsoft.Health.Fhir.Tests.Common;
+using Microsoft.Health.Fhir.Tests.Common.FixtureParameters;
+using Microsoft.Health.Fhir.Tests.E2E.Common;
+using Microsoft.Health.Fhir.Tests.E2E.Rest;
+using Microsoft.Health.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Shared.Tests.E2E.Rest
```

Comment:
namespace should be `Microsoft.Health.Fhir.Tests.E2E.Rest`

Reply:
I comes automatically everytime you create file...
Maybe I'll shoot PR to specify default namespace in shared projects to avoid that (if it's possible ofc)

---
In reply to: [590550086](https://github.com/microsoft/fhir-server/pull/1728#discussion_r590550086) [](ancestors = 590550086)